chess/svg.py

```diff
@@ -130,10 +130,13 @@ def board(board=None, squares=None, flipped=False, coordinates=True, lastmove=No
     :param lastmove: A :class:`chess.Move` to be highlighted.
     :param check: A square to be marked as check.
     :param arrows: A list of :class:`~chess.svg.Arrow` objects like
-        ``[chess.svg.Arrow(chess.E2, chess.E4)]``. Arrows from a square to the
-        same square are drawn as circles.
-    :param size: Limit the SVG size by setting width and height of the image.
-    :param style: CSS to use instead of the default stylesheet.
+        ``[chess.svg.Arrow(chess.E2, chess.E4)]`` or a list of tuples like
+        ``[(chess.E2, chess.E4)]``. An arrow from a square that is pointing to
+        the same square is drawn as a circle, like ``[(chess.E2, chess.E2)]``.
+    :param size: Adjust size of the SVG image by setting its width and
+        height, like ``(600, 600)``.
```

Comment:
*size* is supposed to by just one number, not a tuple, since the board is always quadratic.

Reply:
Well, the docs here says `width and height`, so I figured you need to pass a tuple. Maybe someone wants to have a non-quadratic chessboard size because of space limitation perhaps. Your call on this one.